Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM python:3.7-slim AS pythonapp
+
+ARG PRIMARY_USER=rasgo
+ARG HOME_DIR=/var/lib/rasgo
+ARG UID=1001
+RUN adduser --system --uid ${UID} --home ${HOME_DIR} --shell /bin/false ${PRIMARY_USER}
+
+ARG APP_HOME=${HOME_DIR}/app
+RUN mkdir -p ${APP_HOME} && \
+    mkdir -p ${HOME_DIR}/mount && \
+    chown ${PRIMARY_USER} ${HOME_DIR}
+
+# Dependencies for the Snowflake Python connector
+RUN apt-get update --assume-yes
+RUN apt-get install --assume-yes build-essential libssl-dev libffi-dev
+
+# Configure the entrypoint script
+RUN apt-get install --assume-yes gettext-base # provides envsubst used by next script
+COPY python-entrypoint.sh /usr/bin/entrypoint
+RUN chmod +x /usr/bin/entrypoint
+ENTRYPOINT ["/usr/bin/entrypoint"]
+
+USER ${PRIMARY_USER}
+WORKDIR ${HOME_DIR}
+
+
+FROM pythonapp AS rasgoudt
+ARG HOME_DIR=/var/lib/rasgo
+COPY rasgoudt/requirements.txt ${HOME_DIR}
```

Comment:
do we maybe want the root dir in the container to be called `rasgo_transforms` instead of `rasgoudt` to keep it consistent with Github repo name?

Reply:
yah, good call